lib/rules/sotd/publish.js

```diff
@@ -16,7 +16,7 @@ exports.check = async function (sr, done) {
         docType = `Candidate Registry ${cryType}`;
     }
 
-    const text = `^This document was (?:produced|published) by the (.+? Working Group|Technical Architecture Group|Advisory Board)( and the (.+? Working Group|Technical Architecture Group|Advisory Board))? as a ${docType} using the ${sr.config.track} track.`;
+    const text = `^This document was (?:produced|published) by the (.+? Working Group|Technical Architecture Group( \\(TAG\\))?|Advisory Board|.+? Interest Group)( and the (.+? Working Group|Technical Architecture Group( \\(TAG\\))?|Advisory Board|.+? Interest Group))? as a ${docType} using the ${sr.config.track} track.`;
```

Comment:
Let's keep things simple and not allow acronyms. Otherwise other groups might ask for the same
```suggestion
    const text = `^This document was (?:produced|published) by the (.+? Working Group|Technical Architecture Group|Advisory Board|.+? Interest Group)( and the (.+? Working Group|Technical Architecture Group|Advisory Board|.+? Interest Group))? as a ${docType} using the ${sr.config.track} track.`;
```